fake_joint_trajectory_controller/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(fake_joint_trajectory_controller)
+find_package(catkin REQUIRED)
+catkin_package()
```

Comment:
The same applies as in ros-planning/moveit_robots#44.
Could you please add a `catkin_install_python` install rule for the script?